unit_tests/testVectorTypes/testVectorTypesParallel.f90

```diff
@@ -168,5 +169,109 @@ SUBROUTINE testVector()
       CALL pList%clear()
 
 #endif
-    ENDSUBROUTINE testVector
+    ENDSUBROUTINE testPetscVectorType
+
+    SUBROUTINE testNativeVectorType()
+#if defined(HAVE_MPI)
```

Comment:
Not sure, but do we have style guide requirements for `#ifdefs`?

Reply:
I'm not sure either, but I changed it to be in line with the others.